packages/api-docs-builder/buildApiUtils.ts

```diff
@@ -520,3 +564,137 @@ export const stringifySymbol = (symbol: ts.Symbol, project: TypeScriptProject) =
 
   return formatType(rawType);
 };
+
+export function generateApiPages() {
```

Comment:
Does it make sense to name this function specific to Base? like `generateBaseUIApiPages`?

Reply:
For now yes, let's make it explicit. In the future if we add this to all project, we can make it part of the project's definition.